ui/uihandler.go

```diff
@@ -456,15 +443,12 @@ func LogRequest(r *http.Request) {
 	}
 }
 
-// ChartJSHandler is the handler for the Chart.js library
-func ChartJSHandler(w http.ResponseWriter, r *http.Request) {
-	LogRequest(r)
-	w.Header().Set("Content-Type", "application/javascript; charset=utf-8")
-	w.Header().Set("Cache-Control", "max-age=365000000, immutable")
-	_, err := w.Write([]byte(chartjs))
-	if err != nil {
-		log.Errf("Error writing JS lib to %v: %v", r.RemoteAddr, err)
-	}
+// HTTP handler wrapper to add a Cache-Control header for static files.
+func AddCacheControl(h http.Handler) http.Handler {
+	return http.HandlerFunc (func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Cache-Control", "max-age=3600")
```

Comment:
can you put the same cache-control I had, or why not ?

Reply:
sure